test/support/videoroom_test_helper.ex

```diff
@@ -0,0 +1,41 @@
+defmodule VideoRoomTest.Helper do
+  alias Janus.Plugin.VideoRoom.Errors
+
+  def error_message(error_code, videoroom \\ "event", error \\ "error message") do
+    %{
+      "error_code" => error_code,
+      "videoroom" => videoroom,
+      "error" => error
+    }
+  end
+
+  defmacro test_connection_error(fun, args) do
+    quote do
+      test "returns an error on connection error" do
+        error = {:error, :important_error}
```

Comment:
Are you sure this tests connection error when this tests :important errors?

Reply:
it may be more appropriate to call it session error? The test checks if error returned by `Session.execute_request` is further propagated and returned by `VideoRoom` functions.